parts/21-mutate.sh

```diff
@@ -1408,3 +1408,31 @@ mutate_oidc-by-emails() { ## <email_from> <email_to> [--commit]: Reassign OIDC a
 	txn_pos=$(txn_postfix "$3")
 	QUERY="$txn_pre $QUERY; $txn_pos"
 }
+
+mutate_set_quota_for_oidc_user() { ## <provider_name> <quota_name> [--commit]: Set quota for OIDC users.
```

Comment:
```suggestion
mutate_set_quota_for_oidc_user() { ##? <provider_name> <quota_name> [--commit]: Set quota for OIDC users.
```

this opts into the new cli syntax, where the required parameters are automatically enforced and you don't have to write them any more.